draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -171,6 +163,47 @@
     </section>
   </section>
 
+      <section title="Terminology" anchor="terminology">
+         <t>
+           The following terms are used with special meanings.
+         </t>
+         <t>
+           <list style="hanging">
+                <t hangText="Provider:">
+                   The term "provider" is synonymously with "transit provider", see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                </t>
+
+                <t hangText="Customer:">
+                   An autonomous system uses a provider for transit, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
```

Comment:
perhaps: s/An autonomous system uses a provider for transit/The autonomous system using specified provider for transit/ ?

Reply:
Defining customer and provider is hard because they semantically only make sense as a pair. I think I would prefer keeping it simple and just refer to the aspa-profile draft for a more in depth explanation.